version.php

```diff
@@ -1,3 +1,3 @@
-<?php 
+<?php
 // Please do not modify this file.
```

Comment:
You modified the do not modify file! :P

Reply:
O=)

I still don't actually know whether any of the names matter! But I created a new new project to see what all the defaults were, and then tried to backfill for this project... seems like it worked :)

Reply:
I think it's probably fine in this case, but in my experience "seems like it worked" is a real "hold my beer" of a phrase: it presages disaster.